config/serverless.es.yml

```diff
@@ -15,6 +15,9 @@ enterpriseSearch.enabled: false
 monitoring.ui.enabled: false
 xpack.fleet.enabled: false
 
+# stack monitoring
+xpack.monitoring.enabled: false
```

Comment:
Q: if this is a common setting, why not setting it in `config/serverless.yml` to avoid duplicates? 😇